test/test_beregning.py

```diff
@@ -0,0 +1,13 @@
+
+from fireapi.model import *
+
+def test_beregning(firedb, beregning):
+    firedb.session.commit()
+
+    b0 = firedb.session.query(Beregning).get(beregning.objectid)
+
+    assert b0.objectid == beregning.objectid
```

Comment:
I think we can even go as far as 
```python
assert b0 is beregning
```

And actually that is why `assert b0.objectid == beregning.objectid` works in the first place.

Reply:
mm yes, the tests I made in this PR was in the category "silly stupid" testing.